suites/squid/rgw/tier-2_rgw_test_bucket_notifications.yaml

```diff
@@ -62,26 +62,16 @@ tests:
       name: deploy cluster
 
   # kafka broker type broker with persistent flag enabled
-  - test:
```

Comment:
@anrao19 pls retain this testcase as polarion id is different in different suites

Reply:
suite